exporter/prometheusexporter/collector.go

```diff
@@ -197,11 +199,33 @@ func (c *collector) convertDoubleHistogram(metric pdata.Metric) (prometheus.Metr
 		points[bucket] = cumCount
 	}
 
+	arrLen := ip.Exemplars().Len()
+	exemplars := make([]prometheus.Exemplar, arrLen)
+	for i := 0; i < arrLen; i++ {
+		e := ip.Exemplars().At(i)
+
+		labels := prometheus.Labels{}
```

Comment:
We should allocate up front since we know the length
```suggestion
		labels := make(prometheus.Labels, e.FilteredAttributes().Len())
```